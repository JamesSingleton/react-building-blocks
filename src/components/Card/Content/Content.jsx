import React from 'react';
import PropTypes from 'prop-types';

const Content = ({ children }) => (
  <div>{children}</div>
);

Content.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.node,
    PropTypes.element,
  ]).isRequired,
};

export default Content;
