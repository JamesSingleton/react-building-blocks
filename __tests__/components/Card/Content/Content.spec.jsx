import React from 'react';
import { mount } from 'enzyme';
import Content from '../../../../src/components/Card/Content';

describe('Content', () => {
  it('should render', () => {
    const props = {
      children: 'Hello World',
    };
    const renderedContent = mount(<Content {...props} />);
    expect(renderedContent).toMatchSnapshot();
  });
});
