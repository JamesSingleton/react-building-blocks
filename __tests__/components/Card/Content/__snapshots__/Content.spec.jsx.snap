// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content should render 1`] = `
<Content>
  <div>
    Hello World
  </div>
</Content>
`;
